packages/controls/package.json

```diff
@@ -2,29 +2,38 @@
   "name": "@makeswift/controls",
   "version": "0.0.1",
   "license": "MIT",
-  "type": "module",
   "files": [
     "dist"
   ],
-  "main": "./dist/index.cjs",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
```

Comment:
Changes here and below bring `controls`' `package.json` in line with `prop-controllers`.